include/exadg/grid/mesh_movement_functions.h

```diff
@@ -297,6 +299,48 @@ class RectangleMeshMovementFunctions : public dealii::Function<dim>
   double const                time_period;
 };
 
+
+template<int dim>
+struct RotationMeshMovementData
+{
+  RotationMeshMovementData() : counter_clockwise_degrees_per_s(0.0)
+  {
+  }
+
+  double counter_clockwise_degrees_per_s;
```

Comment:
counter_clockwise is not a good name. If the rotation vector points in positive z/x3-direction, the value should be positive (and else negative).

Please add some documentation.

To extend the usability of this function, the axis around which we rotate should become an additional parameter. It is reasonable in my opinion to limit this code to rotations around one of the dim coordinate axis through the origin.

Reply:
for clarity, I would explicitly write `per_second`.